test/jdk/ProblemList.txt

```diff
@@ -458,6 +458,7 @@ java/awt/Graphics2D/DrawString/RotTransText.java 8316878 linux-all
 java/awt/KeyboardFocusmanager/TypeAhead/ButtonActionKeyTest/ButtonActionKeyTest.java 8257529 windows-x64
 java/awt/KeyboardFocusmanager/ConsumeNextMnemonicKeyTypedTest/ConsumeForModalDialogTest/ConsumeForModalDialogTest.java 8302787 windows-all
 java/awt/KeyboardFocusmanager/TypeAhead/MenuItemActivatedTest/MenuItemActivatedTest.java 8302787 windows-all
+java/awt/KeyboardFocusmanager/ConsumeNextMnemonicKeyTypedTest/ConsumeNextMnemonicKeyTypedTest.java 8321303 linux-all
```

Comment:
This intermittent issue is more reproducible after the fix, however it was there before. 
I am able to reproduce it with various JDK 11, 17, 19, 21, 22 (more info and a reproducer is available [here](https://bugs.openjdk.org/browse/JDK-8321303))

Reply:
This test has been problematic in the past.
Eg https://bugs.openjdk.org/browse/JDK-8283896